pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "pacu"
+version = "0.1.15"
+description = ""
+authors = ["Ryan Gerstenkorn <ryan.gerstenkorn@rhinosecuritylabs.com>"]
```

Comment:
Not really sure what makes sense to be here.. I think this is just what shows up on pip. Might make sense to put pacu@rhinosecuritylabs.com instead.